Wordle.Api/Wordle.Api/Services/PlayerService.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Services;
+
+public class PlayerService
+{
+    public WordleDbContext Db { get; set; }
+    
+    public PlayerService(WordleDbContext db)
+    {
+        Db = db;
+    }
+    
+    public async Task<Player[]> GetTopPlayers(int numberOfPlayers)
+    {
+        return await Db.Players.OrderByDescending(p => p.AverageAttempts).Take(numberOfPlayers).ToArrayAsync();
```

Comment:
I think OrderBy would be the better method to call as lower average attempts is better

Reply:
You are right, thanks